kubernetes/nebula-meta-statefulset.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: nebula
+  name: neubla-metad
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+spec:
+  publishNotReadyAddresses: true
+  clusterIP: None
+  ports:
+  - name: thrift
+    port: 44500
+  - name: raft
+    port: 44501
+  - name: http
+    port: 11000
+  - name: http2
+    port: 11002
+  selector:
+    app: nebula
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: nebula-budget
+  labels:
+    app: nebula
+spec:
+  selector:
+    matchLabels:
+      app: nebula-metad
+  maxUnavailable: 1
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: nebula-metad
+  labels:
+    app: nebula
+spec:
+  serviceName: nebula-metad
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nebula
+  template:
+    metadata:
+      labels:
+        app: nebula
+    spec:
+      terminationGracePeriodSeconds: 60
+      hostname: meta.nebula.services
+      containers:
+      - name: nebula-metad
+        image: vesoft/nebula-metad:nightly
+        imagePullPolicy: IfNotPresent
```

Comment:
the image pull policy should be `Always` for nightly testing since newest docker image always will be tagged `nightly`.

Reply:
ok, i will modify this.